src/main/java/org/folio/marccat/config/constants/Global.java

```diff
@@ -185,6 +186,14 @@
   public static final char BIBLIOGRAPHIC_INDICATOR_NOT_NUMERIC = 'S';
   public static final List<String> FIXED_FIELDS = Arrays.asList("000", "001", "005", "006", "007", "008");
   public static final List<String> MANDATORY_FIELDS = Arrays.asList("000", "001", "008", "040");
+  public static final List<String> AUT_NAMES = Arrays.asList("100", "110", "111");
```

Comment:
Resolve SonarCloud alerts like this for all lists in this class,  for example:
 public static final List<String> AUT_NAMES = Collections.unmodifiableList(Arrays.asList("100", "110", "111"));

Reply:
Thanks. I have changed them.